### DISCUSSION TITLE
Turn tools on/off on course?

### DISCUSSION THREAD

**Discussion by Chawpped** [Courseplay/Courseplay_FS22#1907] [Q&A] [answered]
I've been playing with the course editor and looking to tweak some particularly tricky turns/sections my field. I found that I'm able to create real clean turns using the 'advanced -> curve' tool. But I've run into issues with a number of spots where I would prefer to turn the tool off (planter, lime, fertilizer, whatever) because it's wasting 'ammo' during some of the maneuvers. 

Is there a way for me to tell the course to 'turn off tools' at a certain point, and 'turn on tools' at another?

**Comment by Tensuko** [comment, marked as answer]
Since I don't know how you tweak your turns, I can only tell you the only way to have a nice turn is to use start turn and end turn and not simply create a "turn" with waypoints.
There is no other way to have tools go off and back on and there won't be.

As the help menu writes, we are not responsible for any wrong edited course.

Before you edit your turns that way, you can show us your generated course and the tool and the problem so we might have another solution for you.

**Reply by Chawpped** [reply]
I've just found some of my "official turns" using the turn tool to get stuck from time to time around buildings, obstacles, or tricky terrain/cliffs. Or with some tools the turns are just very slow. 

I can accept that if I want to optimize and customize my turns, I'll just have to be spraying that area with herbicide. No biggy.

Thanks for the response! Happy Farming!
